lib/expression/callbacks/standard.ex

```diff
@@ -116,6 +116,92 @@ defmodule Expression.Callbacks.Standard do
     end
   end
 
+  @expression_doc doc:
+                    ~s[The SWITCH function evaluates one value (called the expression) against a list of values, and returns the result corresponding to the first matching value. If there is no match, an optional default value (the last one in the list if the list is odd) may be returned],
+                  expression: ~s[SWITCH(1, 1, "Sunday", 2, "Monday", 3, "Tuesday", "No match")],
+                  result: "Sunday"
+  @expression_doc doc:
+                    ~s[The SWITCH function evaluates one value (called the expression) against a list of values, and returns the result corresponding to the first matching value. If there is no match, an optional default value (the last one in the list if the list is odd) may be returned],
+                  expression: ~s[SWITCH(5, 1, "Sunday", 2, "Monday", 3, "Tuesday", "No match")],
+                  result: "No match"
+  def switch_vargs(ctx, arguments) do
```

Comment:
thank you! This aligns with the excel function right? https://support.microsoft.com/en-us/office/switch-function-47ab33c0-28ce-4530-8a45-d532ec4aa25e

Reply:
yes indeed, matches "SWITCH" 👍